ktor-io/jvm/src/io/ktor/utils/io/pool/ByteBufferPools.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.utils.io.pool
+
+import io.ktor.utils.io.core.*
+import java.nio.*
+import java.nio.ByteOrder
+
+private const val DEFAULT_POOL_CAPACITY: Int = 2000
+
+private const val DEFAULT_BUFFER_SIZE: Int = 4096
+
+@ExperimentalIoApi
```

Comment:
Don't we want to make them internal (with annotation)?

Reply:
We don't have such for IO, and we can introduce that. I think it's ok to leave it as Experimental, because someone may want to use it in the utility methods.